tests/cypress/integration/reportRepair/contactNumberConfirmation.spec.js

```diff
@@ -32,7 +32,7 @@ describe('contactNumberConfirmation', () => {
     });
     it('Displays the summary box', () => {
       cy.contains('I do not have a number you can call').click()
-      cy.contains('Contact us via telephone on 0800 561 0010')
+      cy.contains(`Contact us via telephone on ${Cypress.env('OUT_OF_HOURS_CUSTOMER_SERVICES_TELEPHONE_NUMBER')}`)
       cy.contains('Our call centre is open between Monday to Friday, 9am to 5pm')
```

Comment:
The explicit opening hours should be replaced with use of the `CUSTOMER_SERVICES_OPENING_HOURS_DESCRIPTION` environment variable (similar to [here](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/blob/main/tests/cypress/integration/landingPageMvp.spec.js#L89))